tests/js/spec/components/externalLink.spec.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import ExternalLink from 'app/components/externalLink';
+
+describe('ExternalLink', function() {
+  it('renders', function() {
+    let wrapper = shallow(<ExternalLink href="https://www.sentry.io/" />);
```

Comment:
can you test the attack here or no?

Reply:
Can't test attack here, the attack vector exists on the destination URL when it has access to `window.opener`